README.md

```diff
@@ -66,21 +74,20 @@ Currently this is only set up for the "h5n1-cattle-outbreak" build, and restrict
 ### To modify this build to work with your own data
 Although the simplest way to generate a custom build is via the quickstart build, you are welcome to clone this repo and use it as a starting point for running your own, local builds if you'd like. The [Nextstrain docs](https://docs.nextstrain.org/en/latest/index.html) are a fantastic resource for getting started with the Nextstrain pipeline, and include some [great tutorials](https://docs.nextstrain.org/en/latest/install.html) to get you started. This build is slightly more complicated than other builds, and has a few custom functions in it to accommodate all the features on [nextstrain.org](https://nextstrain.org/avian-flu), and makes use of wildcards for both subtypes and gene segments. If you'd like to adapt this full, non-simplified pipeline here to your own data (which you may want to do if you also want to annotate clades), you would need to make a few changes and additions:
 
-#### 1. fauna / RethinkDB credentials
-This build starts by pulling sequences from our live [fauna](https://github.com/nextstrain/fauna) database (a RethinkDB
-instance). This requires environment variables `RETHINK_HOST` and `RETHINK_AUTH_KEY` to be
-set.
 
-If you don't have access to our database, you can run the build using the example data
-provided in this repository. Before running the build, copy the example sequences into the
-`data/` directory like so:
+#### 1. Data is stored on a private S3 bucket
+
+The phylogenetics pipeline starts by downloading data from a private S3 bucket.
+If you don't have credentials for this bucket you can run the build using the example data provided in this repository.
+Before running the build, copy the example sequences into the `data/` directory like so:
 
 ```bash
-mkdir data/
-cp example_data/* data/
+mkdir -p data/ha
+cp example_data/metadata_h5n1.tsv data/metadata.tsv
+cp example_data/sequences_h5n1_ha.fasta data/ha/sequences.fasta
```

Comment:
Proposed change in a15fd11.

As I mentioned in that commit's message, the example build needs to run with the `test_target` to avoid trying to build all 48 trees with data from S3.

Reply:
FWIW, the pathogen-repo-guide proposes to use a separate [ci/config.yaml](https://github.com/nextstrain/pathogen-repo-guide/blob/main/phylogenetic/build-configs/ci/config.yaml) that includes a [custom rule](https://github.com/nextstrain/pathogen-repo-guide/blob/6e9e77b94cd7688b4e8a35615a1a06796a4fcf9a/phylogenetic/build-configs/ci/copy_example_data.smk) to copy over the example data.